src/main/resources/assets/securitycraft/lang/ko_kr.json

```diff
@@ -582,15 +659,46 @@
 	"gui.securitycraft:blockPocketManager.activate": "활성화",
 	"gui.securitycraft:blockPocketManager.deactivate": "비활성화",
 	"gui.securitycraft:blockPocketManager.size": "크기: %sx%sx%s",
-
-	"messages.securitycraft:bugReport": "이 GoogleDocs 양식을 사용하여 충돌 또는 버그를 보고할 수 있습니다.",
+	"gui.securitycraft:blockPocketManager.assemble": "자동 조립",
+	"gui.securitycraft:blockPocketManager.outline.show": "윤곽 표시하기",
+	"gui.securitycraft:blockPocketManager.outline.hide": "윤곽 숨기기",
+	"gui.securitycraft:blockPocketManager.youNeed": "필요한 것:",
+	"gui.securitycraft:blockPocketManager.needStorageModule": "블록 포켓의 자동 조립을 활성화하려면 저장 모듈이 필요합니다.",
+	"gui.securitycraft:projector.width": "너비: %s",
+	"gui.securitycraft:projector.width.description": "투영의 너비입니다.",
+	"gui.securitycraft:projector.height": "높이: %s",
+	"gui.securitycraft:projector.height.description": "투영의 높이입니다.",
+	"gui.securitycraft:projector.range": "범위: %s",
+	"gui.securitycraft:projector.range.description": "프로젝터가 자체로부터 블록을 얼마나 멀리 투영할 것인지 나타냅니다.",
+	"gui.securitycraft:projector.offset": "오프셋: %s",
+	"gui.securitycraft:projector.offset.description": "투영의 수평 오프셋입니다.",
+	"gui.securitycraft:projector.vertical": "수직 투영",
+	"gui.securitycraft:projector.horizontal": "수평 투영",
+	"gui.securitycraft:projector.block": "투영할 블록 삽입",
+	"gui.securitycraft:trophy_system.targetableProjectiles": "대상 발사체:",
+	"gui.securitycraft:trophy_system.moduleRequired": "트로피 시스템의 대상이 될 수 있는 발사체 유형을 토글하려면 스마트 모듈을 설치해야 합니다.",
+	"gui.securitycraft:trophy_system.toggle": "특정 발사체 유형의 대상을 전환하려면 아래 목록의 아이템을 클릭하십시오.",
+	"gui.securitycraft:trophy_system.moddedProjectiles": "개량된 발사체",
+	"gui.securitycraft:keycard_reader.signature": "서명: %s",
+	"gui.securitycraft:keycard_reader.keycard_levels": "키카드 단계:",
+	"gui.securitycraft:keycard_reader.link": "링크",
+	"gui.securitycraft:keycard_reader.noSmartModule": "스마트 모듈이 있어야 키카드 단계를 개별적으로 제어할 수 있습니다.",
+	"gui.securitycraft:keycard_reader.smartModule": "개별 키카드 단계 제어가 활성화됩니다!",
+	"gui.securitycraft:keycard_reader.level_mismatch": "이 단계의 키카드를 이 키카드 판독기에서 사용할 수 없습니다!",
+	"gui.securitycraft:customize.tooltip": "%s (%s)",

```

Comment:
This is unchanged to en_us, please remove.

Reply:
You don't have to remove it.